CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v1.3.0
+------------------------------
+*October 5, 2018*
+
+- Added `withinBreakpoint` method to breakpoint helper for breakpoint queries that match our SCSS style
```

Comment:
Need to update to `isWithinBreakpoint` here and on line below

Reply:
done! 👍